{
    "type": "bundle",
    "id": "bundle--90a90509-1dd6-4d1f-912c-55ee0d8c09c0",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--26f5866d-943a-4be4-b37c-bda175e11d0f",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-01-04T00:31:07.57118Z",
            "modified": "2022-01-04T00:31:07.57118Z",
            "name": "CVE-2020-23026",
            "description": "A NULL pointer dereference in the main() function dhry_1.c of dhrystone 2.1 causes a denial of service (DoS).",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2020-23026"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}